superset-frontend/src/explore/components/ExploreChartHeader.jsx

```diff
@@ -205,6 +223,19 @@ export class ExploreChartHeader extends React.PureComponent {
             isRunning={chartStatus === 'loading'}
             status={CHART_STATUS_MAP[chartStatus]}
           />
+          <span
+            role="button"
+            title={t('Schedule email report')}
+            tabIndex={0}
+            className="action-button"
+            onClick={this.showReportModal}
```

Comment:
is @AAfghahi going to add in the logic about permissions and whether the user has a report already or do you want to take it from his code and add it in here?

Reply:
I'm game either way.